mopidy_pandora/library.py

```diff
@@ -145,14 +175,7 @@ def get_images(self, uris):
                             uri
                         )
                     )
-                else:
-                    # Lookup
-                    logger.warning(
-                        "No images available for Pandora URIs of type '{}'.".format(
-                            pandora_uri.uri_type
-                        )
-                    )
-                pass
+
```

Comment:
Given that the amount of exceptions has been reduced, do we need to do the `isinstance` check?